### PR TITLE
Feat/flood first tables form validation

### PR DESCRIPTION
**Pull request by p-chavoshi**
image

### CODE REVIEW

File: apps/frontend/components/DisasterTable/DisasterTable.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  GridColDef,
+  GridColumnGroupingModel,
+  GridRowModel,
+} from '@mui/x-data-grid';
+
+interface IProps {
+  columns: GridColDef[];
+  columnGroup: GridColumnGroupingModel;
+  data: Record<string, string | number | undefined>[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (p-chavoshi) [inline apps/frontend/components/DisasterTable/DisasterTable.tsx:12]: should we figure out event type or is this intentional

File: .eslintrc.js [hunk 2]
```diff
@@ -76,10 +76,6 @@ module.exports = {
       'error',
       {
         paths: [
-          {
```
Comment (p-chavoshi) [inline .eslintrc.js:79]: ?

File: apps/frontend/components/DisasterTable/DisasterTable.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  columns,
+  columnGroup,
+  data,
+  onChange,
+  isEditable,
+}: IProps): JSX.Element => {
+  console;
+
+  return (
+    <div style={{ maxWidth: 1200, height: 240 }}>
```
Comment (p-chavoshi) [inline apps/frontend/components/DisasterTable/DisasterTable.tsx:27]: use 'Box' for consistency

File: apps/frontend/components/DisasterTable/DisasterTable.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+export const DisasterTable = ({
+  columns,
+  columnGroup,
+  data,
+  onChange,
+  isEditable,
+}: IProps): JSX.Element => {
+  console;
```
Comment (p-chavoshi) [inline apps/frontend/components/DisasterTable/DisasterTable.tsx:24]: remove

File: apps/frontend/translations/km.json
Comment (sifelix) [inline apps/frontend/translations/km.json:1]: We are missing the english translation

File: apps/frontend/components/FormValidation/FloodFormValidation/FloodTables.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  isEditMode: boolean;
+}
+
+export const FloodTables = ({
+  value,
+  onChange,
+  isEditMode,
+}: IProps): JSX.Element => {
+  return (
+    <>
```
Comment (p-chavoshi) [inline apps/frontend/components/FormValidation/FloodFormValidation/FloodTables.tsx:27]: do we need the wrapper partial

**Pull request merged**